build.gradle

```diff
@@ -10,11 +11,21 @@ buildscript {
 
 apply plugin: 'com.mparticle.kit'
 
+repositories {
+    maven { url "https://github.com/taplytics/Taplytics-Android-SDK/raw/master/AndroidStudio/" }
+}
+
 dependencies {
-    //TODO: Option 1: Replace with your Maven Central Dependency (preferred)
-    //compile 'com.example:example-artifact:1.2.3'
+    compile 'com.taplytics.sdk:taplytics:+@aar'
```

Comment:
can you use "implementation" instead? Also - this unfortunately cannot be a wildcard. mParticle customers need to have reproducible builds.